sdk/communication/Azure.Communication.Sms/tests/SmsClientRecordedTestSanitizer.cs

```diff
@@ -20,8 +20,7 @@ public override string SanitizeVariable(string variableName, string environmentV
             => variableName switch
             {
                 SmsClientTestEnvironment.ConnectionStringEnvironmentVariableName => SanitizeConnectionString(environmentVariableValue),
-                SmsClientTestEnvironment.FromPhoneNumberEnvironmentVariableName => "+18005551234",
-                SmsClientTestEnvironment.ToPhoneNumberEnvironmentVariableName => "+18005555555",
+                SmsClientTestEnvironment.PhoneNumberEnvironmentVariableName => "+18005555555",
```

Comment:
I remember the discussion that we don't really need 2 numbers for testing this.. but wondering how difficult it would be to actually still have 2 numbers in this test?

Reply:
@ankitarorabit We will need to set up a new number "To" in the key vault in order to use it here. Still don't think we need 2 numbers though since we dont confirm that the SMS has been received on the "To" number